app/labor/badge_rewarder.rb

```diff
@@ -17,15 +17,16 @@ def self.award_yearly_club_badges
   end
 
   def self.award_beloved_comment_badges
+    # ID 3 is the proper ID in prod. We should change in future to ENV var.
```

Comment:
we're slowly switching all these kind of configurations to `SiteConfig`, avoiding env vars, you can check https://github.com/thepracticaldev/dev.to/blob/master/app/models/site_config.rb

I'm not saying it should be done in this PR, just FYI :)

Reply:
In this pr I just moved the line to make the comment more clear. I agree that this should be topic for another pr.